geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -234,8 +257,8 @@ app.DrawController = function($scope, ngeoDecorateInteraction, ngeoLocation,
       function(evt) {
         goog.asserts.assertInstanceof(evt.element, ol.Feature);
         var feature = evt.element;
-        goog.events.listen(feature, goog.events.EventType.CHANGE,
-            this.onFeatureChange_, undefined, this);
+        goog.events.listen(feature, app.DrawEventType.PROPERTYMODIFYEND,
+            this.onFeaturePropertyChange_, undefined, this);
```

Comment:
I'm already using ```ol.ModifyEventType.MODIFYEND```  to manage the geometry change.

My problem with ```ol.ObjectEventType.PROPERTYCHANGE``` is the following : As soon as you modify the feature, an event is sent. 
It sounds great but that means if you change the 'angle' or 'size' with the cursor, a lot of event are sent. 
Then my idea was to send a event only when the user click the OK button to notify that the edition is finished.
I don't know how I can do that with the standard events.

Because when using